src/helpers/apiScripts.js

```diff
@@ -0,0 +1,100 @@
+import axios from "axios";
+
+const apiLink = "https://novi-backend-api-wgsgz.ondigitalocean.app/api/blogposts";
+const apiHeaders = {"novi-education-project-id": "e7334bb5-32f1-462b-a9d0-349bb821c0ba"};
+
+/*---api pull 1.1 ------------------------------------------------------------------------------------------------------*/
+async function pullHelperApi(setError, setApiData, toggleLoading) {
+    setError("");
+    toggleLoading(true);
+
+    try {
+        const response = await axios.get(apiLink, {
+            headers: apiHeaders,
+        });
+        console.log(response);
+        setApiData(response.data);
+    } catch (err) {
+        console.error(err);
+        setError("Er ging iets fout gegaan bij het ophalen van de data.");
+    } finally {
+        toggleLoading(false);
+    }
+}
+
+
+/*---api pull id 1.2 ------------------------------------------------------------------------------------------------------*/
+export async function pullIdNumber(requestId, setError, setApiData, toggleLoading) {
+    setError("");
+    toggleLoading(true);
+
+    try {
+        const response = await axios.get(apiLink, {
+            headers: apiHeaders,
+            params: {"id": requestId}
```

Comment:
Als je een REST API benaderd is het onderdeel van de regels dat als je iets voor een specifieke resource wil doen, zoals een blogpost verwijderen of aanpassen of opvragen, dat dit onderdeel is van het endpoint. Dit is dus **niet** hetzelfde als een request-parameter, die je zo terug ziet komen: https://app/api/blogposts **?naamParam=waarde**. Dit gaat er nu voor zorgen dat de API default naar het ophalen van _alle_ blogposts (array)